integration-test/src/test/java/com/datastrato/gravitino/integration/test/trino/TrinoConnectorIT.java

```diff
@@ -564,6 +564,25 @@ private ColumnDTO[] createFullTypeColumns() {
     };
   }
 
+  @Test
+  void testColumnTypeNotNullByTrino() {
+    String schemaName = GravitinoITUtils.genRandomName("schema").toLowerCase();
+    String tableName = GravitinoITUtils.genRandomName("table").toLowerCase();
+    String createSchemaSql =
+        String.format("CREATE SCHEMA \"%s.%s\".%s", metalakeName, catalogName, schemaName);
+    containerSuite.getTrinoContainer().executeUpdateSQL(createSchemaSql);
+
+    String createTableSql =
+        String.format(
+            "CREATE TABLE \"%s.%s\".%s.%s (id int not null, name varchar not null)",
+            metalakeName, catalogName, schemaName, tableName);
+    containerSuite.getTrinoContainer().executeUpdateSQL(createTableSql);
+
+    catalog
+        .asTableCatalog()
+        .loadTable(NameIdentifier.of(metalakeName, catalogName, schemaName, tableName));
+  }
```

Comment:
Could check the results of the statements of: 
`show create table xxxx` and `insert xxx`

Reply:
OK